packages/tfchain_client/src/utility.ts

```diff
@@ -2,6 +2,7 @@ import { Client } from "./client";
 import { ExtrinsicResult } from "./types";
 import { checkConnection } from "./utils";
 
+const batchSize = 400;

```

Comment:
constants should be uppercase
```suggestion
const BATCH_SIZE = 400;

```